File: app/src/main/java/com/imangazalievm/bubbble/presentation/mvp/main/MainPresenter.java
package com.imangazalievm.bubbble.presentation.mvp.main;

import com.arellomobile.mvp.InjectViewState;
import com.arellomobile.mvp.MvpPresenter;
import com.imangazalievm.bubbble.presentation.mvp.main.MainView;

@InjectViewState
public class MainPresenter extends MvpPresenter<MainView> {

    public MainPresenter() {
    }

    public void onSearchQuery(String searchQuery) {
        getViewState().openSearchScreen(searchQuery);
    }

}

File: app-mvp/app/src/main/java/com/bubbble/presentation/main/MainPresenter.kt
package com.bubbble.presentation.main

import com.arellomobile.mvp.InjectViewState
import com.arellomobile.mvp.MvpPresenter

@InjectViewState
class MainPresenter : MvpPresenter<MainView>() {

    fun onSearchQuery(searchQuery: String) {
        viewState.openSearchScreen(searchQuery)
    }

}